content/templates/implementations.md

```diff
@@ -5,102 +5,110 @@ description: "A curated list of awesome things related to Lottie"
+- <a href="https://github.com/simolus3/fluttie" target="_blank">Fluttie</a> (Flutter) by Simon Binder
 
 ##### iOS
-
 - <a href="https://github.com/airbnb/lottie-ios" target="_blank">lottie-ios</a> by Airbnb
-- <a href="https://github.com/airbnb/lottie-react-native" target="_blank">lottie-react-native</a> by Airbnb
-- <a href="https://github.com/LottieFiles/dotlottie-ios" target="_blank">dotlottie-ios</a>
-
-##### Desktop
+- <a href="https://github.com/airbnb/lottie-react-native" target="_blank">lottie-react-native</a> (React Native) by Airbnb
+- <a href="https://github.com/LottieFiles/dotlottie-ios" target="_blank">dotlottie-ios</a> by LottieFiles
 
-- <a href="https://github.com/windows-toolkit/Lottie-Windows" target="_blank">lottie-windows</a> by Microsoft
-- <a href="https://github.com/azchohfi/LottieUWP" target="_blank">Lottie for UWP</a>
-- <a href="https://lottiefiles.com/desktop" target="_blank">LottieFiles Desktop</a>
+##### Windows
+- <a href="https://github.com/windows-toolkit/Lottie-Windows" target="_blank">Lottie-Windows</a> by Microsoft
+- <a href="https://github.com/azchohfi/LottieUWP" target="_blank">LottieUWP</a> (Universal Windows Platform) by Alexandre Zollinger Chohfi
 
 ##### Multi-Platform
-
-- <a href="https://www.qt.io/blog/2019/03/08/announcing-qtlottie" target="_blank">Qt</a>
-- <a href="https://skia.org/docs/user/modules/skottie/" target="_blank">Skottie</a> - Lottie Animation Player by Google
-- <a href="https://www.thorvg.org/" target="_blank">Thorvg</a> - for rendering vector-based scenes and animations, including SVG and Lottie formats
-- <a href="https://github.com/Samsung/rlottie" target="_blank">rlottie</a> - by Samsung
+- <a href="https://doc.qt.io/qt-6/qtlottieanimation-index.html" target="_blank">Qt Lottie Animation</a> (Qt)
+- <a href="https://skia.org/docs/user/modules/skottie/" target="_blank">Skottie</a> (Skia) by Google
+- <a href="https://github.com/Samsung/rlottie" target="_blank">rlottie</a> (C++) by Samsung
+- <a href="https://github.com/LottieFiles/dotlottie-rs" target="_blank">dotlottie-rs</a> (Rust) by LottieFiles
+- <a href="https://www.thorvg.org/" target="_blank">ThorVG</a> - for rendering vector-based scenes and animations, including SVG and Lottie formats
 
 <br/>
 
-## Tools
+## Viewers
+- <a href="https://www.lottielab.com/preview" target="_blank">Lottielab Preview</a> (Web)
+- <a href="https://app.lottiefiles.com/preview" target="_blank">LottieFiles Preview</a> (Web)
+- <a href="https://lottiefiles.com/desktop" target="_blank">LottieFiles Desktop</a> (Mac)
+- <a href="https://risefall.org/#app-inari" target="_blank">Inari</a> (Mac, Windows)
+- <a href="https://www.thorvg.org/viewer" target="_blank">ThorVG Viewer</a> (Web)
+- <a href="https://drawer.design/preview-lottie-animation/" target="_blank">Drawer Preview</a> (Web)
+- <a href="https://lottie.cloud" target="_blank">LottieCloud</a> (Web, iOS)
```

Comment:
```suggestion
```
The site and app look very laggy and untrustworthy, with a low App Store ranking. Additionally, it seems like the tool is no longer being maintained.